pkgs/applications/networking/shellhub-agent/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "shellhub-agent";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "shellhub-io";
+    repo = "shellhub";
+    sha256 = "0cd41ing1pcf1bdaaq00w5h7lih5j2kcaa0m41g3ikm3vd1w5qna";
+  };
+
+  patches = [ ./go-mod-tidy.patch ];
+
+  vendorSha256 = "1k4i6jvf7xivk3ahsd7a1lj3s6rzdanig50qpc6rx4swvcidi2pa";
+
+  preConfigure = "cd agent";
+
+  buildFlagsArray = [ "-ldflags=-s -w -X main.AgentVersion=v${version}" ];
+
+  meta = with stdenv.lib; {
+    description =
```

Comment:
The text is pretty long and fits better into longDescription. Can you add a shorter one which does not exceed about 80 characters?

Reply:
Sure; I'll rework it.